lib/LaTeXML/Post.pm

```diff
@@ -524,8 +524,9 @@ sub associateNode {
       $op = undef; }
     elsif ($q eq 'ltx:XMTok') { }
     elsif ($q eq 'ltx:XMApp') {
-      ($op) = element_nodes($op);
-      $q = $document->getQName($op) || 'unknown'; }    # get "real" operator
+      while ($q eq 'ltx:XMApp') {
```

Comment:
I would argue this may be mildly more annoying, if you encounter an `XMApp` with a set meaning attribute, say `meaning=decorated-arrow-operator` or some such, where the association is really with that application, rather than the deepest token (which would be a regular arrow in my hypothetical, without its embellishments).

Reply:
Yeah, probably should stop the iteration if there's a `meaning`.